homeseer/hs-device.html

```diff
@@ -187,6 +187,11 @@ <h3>Input</h3>
 		<dt class="optional">msg.payload.status <span class="property-type">string</span></dt>
 		<dd>the status text of the control you want to use</dd>
 	</dl>
+    <p>To update a device string use <b>msg.payload.string</b></p>
```

Comment:
The whole help section needs to be updated to inform user he needs to use msg.topic with either control or update

Reply:
This is on my todo pending your thumbs up on the other changes. I'll have something soon.